### PR TITLE
[10.0][FIX] Misc fixes

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Comment by acsonefho** [comment]
Commit about product_multi_link is about this PR who breaks related shopinvader module: https://github.com/OCA/e-commerce/pull/452

File: shopinvader_product_template_multi_link/models/shopinvader_product.py [hunk 1]
```diff
@@ -26,31 +26,36 @@ class ShopinvaderProduct(models.Model):
...
-                    binding.backend_id == current_backend
-                    and binding.lang_id == self.lang_id
-                ):
-                    # Set only the "main" shopinvader variant
-                    for shopinvader_variant in binding.shopinvader_variant_ids:
-                        if shopinvader_variant.main:
-                            res.append(shopinvader_variant.id)
-                            break
-        return res
+        product_links = self.product_template_link_ids.filtered(
```
Comment (simahawk) [inline shopinvader_product_template_multi_link/models/shopinvader_product.py:31]: I'd recommend to backport the current status of v13 unless you see any real problem w/ it

**Review by simahawk — COMMENTED** [review]
maybe change the title? Seems not to be "just" a fix for requirements :wink:

**Comment by acsonefho** [comment]
> maybe change the title? Seems not to be "just" a fix for requirements

Yes but as I'm not the author I can't edit it =/

File: shopinvader_product_template_multi_link/models/shopinvader_product.py (hunk 1 above)
Comment (acsonefho) [reply shopinvader_product_template_multi_link/models/shopinvader_product.py:31]: I just fix errors with unit test for v10.0 (the main goal of this PR). I never use this module.
IMO this backport should be done into a new dedicated PR.

**Comment by lmignon** [comment]
Closing since 10.0 is no more actively maintained.

**Pull request closed without merging**